packages/limit-orders-lib/src/constants/index.ts

```diff
@@ -15,6 +15,7 @@ export const CHAIN_ID = {
 };
 
 export const L2_BPS_GELATO_FEE = {
```

Comment:
we should rename this constant, removing `L2` from it since it is now a global fee on all networks (including ethereum)